### DISCUSSION TITLE
How to update github repo using golem?

### DISCUSSION THREAD

**Discussion by Dobrokhotov1989** [ThinkR-open/golem#681] [Q&A] [answered]
This is more a question than an issue.

I'm building my first shiny app using `{golem}`. In 02_dev.R, I used `usethis::use_github()` to create Github repo and made initial commit.  Now I'd like to push updated versions of files into Github repo. How can I do it from `{golem}`?

**Comment by ColinFay** [comment, marked as answer]
Hey @Dobrokhotov1989, 

You can push to your repository using the following commands in your terminal: 

```r
git add *
git commit -m "message"
git push origin master
```

You can also use a package like `{git2r}` to do this, or via the RStudio Git panel. 

You can read this very good guide about git and R if you want to learn more :) 

https://happygitwithr.com/

Cheers, 
Colin
